content/en/tracing/setup_overview/setup/dotnet-framework.md

```diff
@@ -358,7 +358,13 @@ Your logger needs to have a `source` that sets the `trace_id` mapping correctly.
 
 `DD_TRACE_SAMPLE_RATE`
 : **TracerSettings property**: `GlobalSamplingRate` <br>
-Enables ingestion rate control.
+**Default**: `1.0`<br>
+Enables ingestion rate control. The sampling rate for the traces (defaults to: between `0.0` and `1.0`).
+
+`DD_TRACE_SAMPLING_RULES`
+: **TracerSettings property**: `datadog.trace.sampling_rules`<br>
+**Default**: `null`<br>
+A JSON encoded string to configure the sampling rate. Examples: Set the sample rate to 20%: `'[{"sample_rate": 0.2}]'`. Set the sample rate to 10% for services starting with 'a' and span name 'b' and set the sample rate to 20% for all other services: `'[{"service": "a.*", "name": "b", "sample_rate": 0.1}, {"sample_rate": 0.2}]'` (see [Integration names](#integration-names)). Note that the JSON object **must** be included in single quotes (`'`) to avoid problems with escaping of the double quote (`"`) character.|
```

Comment:
```suggestion
A JSON encoded string to configure the sampling rate. Examples: Set the sample rate to 20%: `'[{"sample_rate": 0.2}]'`. Set the sample rate to 10% for services starting with 'a' and span name 'b' and set the sample rate to 20% for all other services: `'[{"service": "a.*", "name": "b", "sample_rate": 0.1}, {"sample_rate": 0.2}]'` (see [Integration names](#integration-names)). Note that the JSON object **must** be included in single quotes (`'`) to avoid problems with escaping of the double quote (`"`) character.
```